src/command_modules/azure-cli-container/README.rst

```diff
@@ -48,8 +48,8 @@ Commands to create an Azure container group
         --registry-username           : The username to log in container image registry server.
 
     Log Analytics Arguments
-        --log-analytics-workspace       : The Log Analytics workspace. Must be in the currently set subscription.
-        --log-analytics-workspace-id    : The Log Analytics workspace id.
+        --log-analytics-workspace       : The Log Analytics workspace name or id. If a name is
+                                        specified, it must be within the currently set subscription.
```

Comment:
Note this is not true any longer. The global `--subscription` parameter has been added, so you could use names and still set a different subscription provided your name-or-id logic is following the accepted patterns..